packages/api/test.tap

```diff
@@ -0,0 +1,109 @@
+1..76
+Example app listening at http://localhost:3000
+ok 1 Add-Build add-build Can add a build and repository to a blank collection
+BUIDLS>DATA: 1
+ok 2 Add-Build add-build Can add a different build with overlapping test cases and update repo info
+ok 3 Add-Build add-build Can add a third build with some new test cases, add same thing multiple times
```

Comment:
we shouldn't check in these tap files, not quite sure what happened here, as I added it to the `.gitignore`, I perhaps screwed up  the `.gitignore` directive.

Mind removing this with `git rm --force packages/api/test.tap`.